src/realm/util/file.cpp

```diff
@@ -1587,7 +1622,13 @@ std::string File::get_path() const
 
 std::string File::resolve(const std::string& path, const std::string& base_dir)
 {
-#ifndef _WIN32
+#if REALM_HAVE_STD_FILESYSTEM
+    std::filesystem::path path_(path.empty() ? "." : path);
+    if (path_.is_absolute())
+        return path;
```

Comment:
Isn't this included in the behavior of `path`'s `operator<`? If this is different and needed for some reason, a comment explaining why would be nice.

Reply:
You're absolutely right. According to https://github.com/microsoft/STL/blob/73924c1920af92899f7582cd904ea819b9db35bc/stl/inc/filesystem#L708-L720 that's indeed the case. Side note: I love that Microsoft's STL is open source now and I can get useful information about its workings like this.

Reply:
FYI, I just looked it up at https://en.cppreference.com/w/cpp/filesystem/path/append (`operator/` docs just say to see here for behavior)